redaxo/src/core/lib/packages/manager.php

```diff
@@ -479,6 +479,21 @@ public function checkConflicts()
             }
         }
 
+        foreach (rex_package::getAvailablePackages() as $package) {
```

Comment:
$e = [];
    $e['label'] = '<label for="rex-mediapool-category">' . rex_i18n::msg('pool_file_category') . '</label>';
    $e['field'] = $cats_sel->get();
    $formElements[] = $e;